src/Raven.Client/Http/ClusterRequestExecutor.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Client.Server.Commands;
+using Sparrow.Json;
+
+namespace Raven.Client.Http
+{
+    public class ClusterRequestExecutor : RequestExecutor
+    {
+
+        protected ClusterRequestExecutor(string databaseName, string apiKey) : base(databaseName, apiKey)
+        {
+        }
+
+        public new static ClusterRequestExecutor Create(string[] urls, string databaseName, string apiKey)
```

Comment:
Why do we have  database name when we are talking about cluster request manager?